src/test/system_tests/test_ceph_s3.js

```diff
@@ -1,631 +0,0 @@
-/* Copyright (C) 2016 NooBaa */
```

Comment:
This file was renamed...
This was my git status before I amended this commit:
![Screen Shot 2023-01-02 at 19 04 53](https://user-images.githubusercontent.com/57721533/210261391-e87760a3-2c88-4faf-a47b-1a7e1f6e9502.png)